File: C#/백준/Bronze/30979. 유치원생 파댕이 돌보기/유치원생 파댕이 돌보기.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int b = int.Parse(Console.ReadLine());
            string[] cc = Console.ReadLine().Split();
            int[] aa = Array.ConvertAll(cc,int.Parse);
            if(a<=aa.Sum()){
                Console.WriteLine("Padaeng_i Happy");
            }
            else{
                Console.WriteLine("Padaeng_i Cry");
            }
        }
    }
}

File: Java/백준/Bronze/30979. 유치원생 파댕이 돌보기/유치원생 파댕이 돌보기.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int b = in.nextInt();
        int sum = 0;
        for (int i =0; i<b; i++){
            sum+=in.nextInt();
        }
        if(sum>=a){
            System.out.printf("Padaeng_i Happy");
        }
        else{
            System.out.printf("Padaeng_i Cry");
        }
    }
}